rclpy/rclpy/__init__.py

```diff
@@ -82,7 +82,7 @@ def spin_once(node, timeout_sec=None):
             )
 
     if timeout_sec is None:
-        timeout = -1
+        timeout = S_TO_NS
```

Comment:
I know it's the same thing, but I think it is more readable to do `1 * S_TO_NS`.